src/documentation/io/testing.md

```diff
@@ -179,40 +179,46 @@ Parameter descriptions:
   </thead>
   <tbody>
     <tr>
-     <td>-Dio-it-suite
+     <td>-DpkbLocation
      </td>
-     <td>Invokes the call to PerfKit Benchmarker when running in apache beam's jenkins instance or with a kubernetes cluster on the same network.
+     <td>Path to PerfKit Benchmarker project.
      </td>
     </tr>
     <tr>
-     <td>-Dio-it-suite-local
+     <td>integrationTestPipelineOptions
```

Comment:
Should this one have -D like the other options?

Reply:
True - same as `logLevel` in line 450. Corrected both.